### DISCUSSION TITLE
Whether there is a method to get the relative position between two objects

### DISCUSSION THREAD

**Discussion by lykl** [pixijs/pixijs#8998] [Q&A]
For example, I have elements of the following format
This `container` has displacement and rotation
```
-stage
  -container
    -graphics
  -graphics
```
What do I do if I want to get the position of the `graphics` outside the `container` relative to the `graphics` inside the `container`(this location is the location inside the `container`)
Or if I register a `mousedown` event for `stage`, I want to get the position of the mouse click inside the `container`, and if I click outside the `container`, of course the `container` is shifted and rotated.

**Comment by miltoncandelero** [comment]
you can try getting the `toGlobal` position for both by doing `graphics.toGlobal(new Point())` and that will return the position of an object in global coordinates. Then just measure the distance between those two global points

**Reply by lykl** [reply]
What I get this way is the global coordinates, but I want to know the coordinates of the external object or the mouse click position relative to the container

**Reply by lykl** [reply]
If the container is not rotated, the situation is not very complicated, and a series of conversions will be OK. But when it comes to rotation, the situation will be very complicated. Although I try to calculate, the value is always wrong, which makes me feel very weak, maybe my math is not good
